### ISSUE TITLE
libs have no SONAME

### ISSUE THREAD

**Issue by tjaalton** [intel/intel-graphics-compiler#51]
The produced libs have no SONAME, please fix that so that distros can ship proper packages.

**Comment by chivakker** [comment]
Please do fix this,  but also please explain how you're going to make compatibility work with different versions of dependencies, it would be good if you guys can describe the full picture.

**Comment by iwwu** [comment]
Among our list of .so libraries, libcommon_clang.so has SONAME set. I'll look into adding this field to other libraries
libiga.so
libigc.so
libigdfcl.so
libopencl_clang.so

**Comment by nandojve** [comment]
I had problems to build NEO because of IGC libraries.

1) When a shared library isn't found, *ocloc* abort without any error message. Please, when load shared library check properly and print/log error message to help users.
2) Library names isn't right: intel/compute-runtime#133
Please, use the right denomination to all libraries:

```
LINK          22 Feb 19 21:13 libcommon_clang.so -> libcommon_clang.so.7.1
SONAME        22 Feb 19 21:13 libcommon_clang.so.7 -> libcommon_clang.so.7.1
LIBRARY 73373504 Feb 18 20:17 libcommon_clang.so.7.1
```

I believe this small improvement could help a lot.

**Comment by tripzero** [comment]
https://github.com/intel/intel-graphics-compiler/pull/74 fixes this issue.

**Comment by paigeale** [comment]
Closing. Issue fixed with 3cc1170208f6a0c984e906fc5d531798313d5f40
